core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -519,6 +542,7 @@ public FileInfo setMountId(long mountId) {
         mInMemoryPercentage, mLastModificationTimeMs, mTtl, mOwner, mGroup, mMode,
         mPersistenceState, mMountPoint, fileBlockInfos, ThriftUtils.toThrift(mTtlAction), mMountId,
         mInAlluxioPercentage);
+    info.setUfsFingerprint(mUfsFingerprint);
```

Comment:
This cannot be passed in the constructor?

Reply:
No, because `ufsFingerprint` is not a required field. I was under the impression that new thrift fields should be optional to be compatible. If we are ok with making it required, then it would be in the constructor.

Reply:
I changed it to default. Thanks!

Reply:
I changed it to default.